src/app/zwiazek/historia-zwiazku/data.ts

```diff
@@ -0,0 +1,51 @@
+interface Paragraph {
+  paragraph: string | string[];
+}
+export interface Subsection extends Paragraph {
+  subtitle?: string;
```

Comment:
Make two types.
```tsx
interface A extends Paragraph {
    subtitle: string;
}

interface B extends A {
    bulletpoints: string[];
}

// or
interface B_version2 extends Paragraph {
    subtitle: string;
    bulletpoints: string[];
}
```

Reply:
Don't be afraid to make specialised interfaces and data shapes. It is not always worth it to make it fit one shape.
DRY is important but do not forget about WET 😉 

We could even make them totally separate and not extend each other. Although it is such a simple case that we can extend them, in case of some changes it will be easy to implement.